Dockerfile

```diff
@@ -9,11 +9,26 @@ RUN corepack prepare pnpm@latest --activate
 RUN pnpm install
 RUN pnpm build
 
-FROM caddy:alpine
+FROM golang:1.20-bullseye as build-bundle
 
-WORKDIR /srv
+WORKDIR /build
+
+COPY --from=build-web /build/dist web
+
+RUN apt-get update && apt-get install -y llvm clang git make
+RUN git clone https://github.com/daeuniverse/dae-wing.git
+RUN cd dae-wing && make OUTPUT=daed WEB_DIST=/build/web/ bundle
```

Comment:
To consider cross-image compatibility, CGO_ENALBED=0 is necessary in most cases, or just keep in the same image with builder and remove useless pkgs.